docs/Documento_metodologia.md

```diff
@@ -0,0 +1,89 @@
+# Documento de Metodologia e Processos
+
+## Histórico
+|Data|Versão|Modificação| Autor(es)
+|--|--|--|--|
+|17/02/2021|1.0|Abertura do Documento|Vitor Lamego e Victor Lima|  
+|17/02/2021|1.1|Descrição das metodologias|Vitor Lamego|
+
+## 1. Introdução
+Este documento possui a finalidade de descrever e explicar quais serão as metodologias utilizadas no projeto. Será feito uma breve introdução das metodologias que serão utilizadas, assim como uma apresentação de quais artefatos e rituais/cerimônias serão realizadas pela equipe.
+
+## 2. Metodologias de Base
+A equipe não irá utilizar somente uma metodologia dentro do projeto, sendo então adotada uma metodologia híbrida, com base no SCRUM, Kanban e XP. A ideia é reunir alguns artefatos e rituais dessas diferentes metodologias e colocá-las em prática no andamento do projeto.
+
+Acreditamos que a utilização de uma metodologia híbrida possa ajudar na oferta de uma maior experiência e contato com diferentes processos já consolidados no mercado, além disso o objetivo da equipe não é estar preso a um processo irreversível, e sim estar apto a conhecer coisas novas e estar disposto a aplicar mudanças dentro do processo de acordo com o que a equipe sentir necessidade. "Respostas às mudanças do que seguir um contrato", como diz um dos valores das Metodologias Ágeis.
+
+### 2.1 Scrum
+O SCRUM é uma metodologia ágil muito conhecida e que já até se expandiu para diversas outras áreas além da Engenharia de Software. Focada no planejamento e gestão de projetos, ocorre de maneira iterativa, onde a iteração é mais conhecida por **Sprint**, que consiste no período que pode variar de 1 a 4 semanas. Dentro do período da Sprint os membros da equipe devem exercer as tarefas que assim foram demandadas naquele período de tempo.
+
+O SCRUM possui alguns artefatos e rituais que colaboram de maneira significativa para o projeto. Durante a Sprint são realizadas reuniões diariamente conhecidas por Daily Meeting, que servem para alinhar toda a equipe do que vem acontecendo dentro do projeto, são reuniões rápidas e com o sentimento de urgência para que não seja extensa. Outros artefatos muito utilizados e falado no SCRUM são o **Product Backlog** que consiste em uma lista com todas as funcionalidades desejadas para o produto, e também o **Sprint Backlog** que se destina a todas as issues que a equipe se compromete a fazer naquela Sprint.
+
+Além disso, o SCRUM define três papéis dentro da equipe, sendo eles:
+* **Scrum Master**: Responsável por garantir que os valores e práticas do SCRUM estão sendo aplicados pela equipe. Também é responsável por assegurar que a equipe não se comprometa excessivamente durante uma Sprint.
```

Comment:
Aqui ele não está pulando a linha como esperado, acho que faltou um enter entre a linha anterior e essa.

Reply:
Corrigindo, ali na linha 23 para baixo. Marcou errado e eu não sei porque.

Reply:
Você diz ali na parte que ele começa a pontuar os papéis ?? No meu depois do "sendo eles:" ele está pulando linha normalmente, caso seja isso

Reply:
Que estranho, para mim ta assim quando eu rodo o mkdocs
![image](https://user-images.githubusercontent.com/46077033/108400384-0a702980-7213-11eb-9436-1a4fff37653c.png)

Reply:
Ok, de qualquer forma acrescentei mais um espaço para garantir.